stack.yaml

```
@@ -0,0 +1,8 @@
+snapshot:
+  url: https://raw.githubusercontent.com/commercialhaskell/stackage-snapshots/master/lts/23/28.yaml
+
+packages: [.]
+
+nix:
+  enable: true
+  packages: [zlib.dev, zlib.out, pkg-config]
```

I assume this didn't work?
```suggestion
  packages: [zlib, pkg-config]
```

---

I didn't try this. I'll do that and update